### PR TITLE
FIX: Add theme skeleton

### PR DESCRIPTION
**Pull request by italoaalves**
This PR includes the theme skeleton to this repository ensuring it will comply with our standard plugin/themes format.

### CODE REVIEW

File: settings.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+example_setting:
```
Comment (pmusaraj) [inline settings.yml:1]: This file you don't need. It'll add a setting in the UI.

**Review by pmusaraj — COMMENTED** [review]
Can you also add the hidden `.github` folder? It will make sure that the linting checks run for PRs.

File: settings.yml (hunk 1 above)
Comment (italoaalves) [reply settings.yml]: Done

**Comment by pmusaraj** [comment]
Awesome, thanks, almost there! Linting is failing, both for Prettier and for ESLint. You can fix this by configuring your editor to auto-lint, or by running this from the root of the theme folder: 

```
yarn run prettier --write javascripts/discourse/lib/iecst-highlightjs.js
yarn run prettier --write javascripts/discourse/initializers/iecst.js
yarn run prettier --write javascripts/discourse/lib/iecst-highlightjs.js
```

That's for prettier. For ESlint, the error raised is: 

```
/home/runner/work/discourse-highlightjs-structured-text/discourse-highlightjs-structured-text/javascripts/discourse/initializers/iecst.js
Error:   6:14  error  'container' is defined but never used  no-unused-vars
```

Since you're not using the `container` variable, you can just skip it, on https://github.com/discourse/discourse-highlightjs-structured-text/blob/main/javascripts/discourse/initializers/iecst.js#L6, convert it to `initialize() {`.

**Review by pmusaraj — APPROVED** [review]
Sweet, checks are all green, thanks!

**Pull request merged**